### ISSUE TITLE
[Ember 1.13.0] setDate not working ('this' is in state 'preRender')

### ISSUE THREAD

**Issue by DenizOkcu** [edgycircle/ember-pikaday#25]
Hi David,
[setDate()](https://github.com/edgycircle/ember-pikaday/blob/master/addon/components/pikaday-input.js#L55) is not working with the new ember beta (1.13.0-beta.1 incl. Glimmer).
```javascript
setDate: function() {
  this.get('pikaday').setDate(this.get('value'), true);
}.observes('value'),
```
`this` is in state 'preRender' at that point in time what makes `pikaday` undefined.

`Uncaught TypeError: Cannot read property 'setDate' of undefined`

We could fix it like this:
```javascript
setDate: function() {
  Ember.run.scheduleOnce('afterRender', this, function() {
    this.get('pikaday').setDate(this.get('value'), true);
  });
}.observes('value'),
```
Tests are green. Other uses of `get('pikaday')` don't seem affected. What do you think?

**Comment by DenizOkcu** [comment]
ok tests are only OK in the browser, 13 tests are failing on the command line :-/

**Comment by stravid** [comment]
That's a bummer, thanks for looking into this. I think there should be a new Ember 1.13.0 beta tomorrow, maybe this improves things.

**Comment by stravid** [comment]
@DenizOkcu is this still an issue?

**Comment by polymathnyc** [comment]
I'm still seeing this issue with Ember 1.13.0. The fix from @DenizOkcu seems to solve the issue however.

**Comment by stravid** [comment]
@polymathnyc Thanks for reporting, I ran the tests against Ember 1.13.0 and they where green so I assumed this issue went away. I will take a look and try to reproduce and fix it!

**Comment by stravid** [comment]
This should be fixed in version `0.7.2`.
